letsdrill.rb

```diff
@@ -1,13 +1,25 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+ 	if integer >= 90
+  		"A"
+ 	elsif integer >= 80
+ 		"B"
+ 	elsif integer >= 70
+ 		"C"
+ 	elsif integer >= 60
+ 		"D"
+ 	else
+ 		"F"
+ 	end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  shortest = nil
```

Comment:
This method works but you should consider looking into ruby's Enumerable methods. `.min_by` & `sort_by` are good options here.